assets/queries/dockerfile/disable_pip_cache/query.rego

```diff
@@ -0,0 +1,29 @@
+package Cx
+
+CxPolicy [ result ] {
+	resource := input.document[i].command[name][_]
+
+  resource.Cmd == "run"
+ 	
+  values := resource.Value[0]
+    
+  hasCacheFlag(values)
+    
+	result := {
+    			    "documentId": 		input.document[i].id,
+              "searchKey": 	    sprintf("FROM={{%s}}.RUN={{%s}}", [name, resource.Value[0]]),
+              "issueType":		 "IncorrectValue",
+              "keyExpectedValue": "The '--no-cache-dir' flag is set when running 'pip install'",
+              "keyActualValue": 	 "The '--no-cache-dir' flag isn't set when running 'pip install'",
+            }
+}
+
+hasCacheFlag(values){
+	commands = split(values,"&&")
+    
+	some i
+    	instruction = trim_space(commands[i])
+      startswith(instruction, "pip")
+      contains(instruction, "install")
+      not contains(instruction, "--no-cache-dir")
```

Comment:
The query is missing check for, "python -m pip install"

Reply:
Fixed in https://github.com/Checkmarx/kics/pull/1410